pbsmetrics/prometheus/prometheus.go

```diff
@@ -63,6 +70,10 @@ const (
 	requestStatusLabel   = "request_status"
 	requestTypeLabel     = "request_type"
 	successLabel         = "success"
+
+	adapterConnErrorLabel   = "adapter_connection_error"
```

Comment:
Since you've defined separate metrics for `ConnError`, `ConnReused` and `ConnCreated`, any reason why we also need labels? Also, they don't seem to be used anywhere.

Reply:
Pretty good catch. This is a leftover from the previous commit. Removed